mimic/test/test_loadbalancer.py

```diff
@@ -109,6 +110,31 @@ def _create_loadbalancer(self, name=None, api_helper=None):
         create_lb_response_body = self.successResultOf(treq.json_content(create_lb_response))
         return create_lb_response_body['loadBalancer']['id']
 
+    def test_lb_status_changes_as_requested(self):
+        """
+        Perhaps a bit mislabeled, this test exercises the endpoint that causes
```

Comment:
We try hard in mimic to have descriptive test docstrings, that assert something positive about the system under test.  Don't apologize for the mislabeling, just re-label it :).

Particularly, we try to avoid words like "process", "handle", "exercise", "correct", "test", "valid", "execute" etc, since all of these are ways to avoid describing what is happening directly and instead say *that* something is happening.

But to give you an example what I mean, something like this, for this test:

```python
        """
        Clients can ``POST`` to the ``.../loadbalancer/<lb-id>/setAttr``
        :obj:`LoadBalancerControlApi` endpoint in order to change the
        ``status`` attribute on the load balancer identified by the given
        load-balancer ID for the same tenant in the :obj:`LoadBalancerApi`.

        This attribute controls the status code returned when the load balancer
        is retrieved by a ``GET`` request.
        """
```